Rubberduck.Resources/Templates.resx

```diff
@@ -130,9 +130,8 @@ Attribute VB_Creatable = False
 Attribute VB_PredeclaredId = True
 Attribute VB_Exposed = False
 Attribute VB_Ext_KEY = "Rubberduck", "Predeclared Class Module"
-
-Option Explicit
-'@PredeclaredId</value>
+'@PredeclaredId
+Option Explicit</value>
```

Comment:
There is a coded set of header attributes in [`ExtractInterfaceRefactoringAction`](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceRefactoringAction.cs) if the option is selected to expose the interface.  I'm wondering if both the `ExtractInterfaceRefactoringAction` string and this one belong in a common resource file.   If so, I would have put the `ExtractInterface` related text in `Refactorings` as `Templates` does not seem right.  Yet `Refactorings` does not seem the correct 'grouping' for this one either...thoughts?

Reply:
Correct, that would be a different point of the #4549 which is not addressed in this PR. In order to extract all the code snippets out of resources file and into the plain text, we need a convention or a macro system to describe how to use the code snippets to allow user customization of those code snippets. Note that even if the #4549 was fully implemented, we'd still have the original code snippets in the resource files as they need to be able to be reconstructed in event the text files get deleted or is unusable to avoid the duck from going down in flames.